Modelica/Thermal/HeatTransfer/UsersGuide/References.mo

```diff
@@ -0,0 +1,24 @@
+within Modelica.Thermal.HeatTransfer.UsersGuide;
+class References "References"
+  extends Modelica.Icons.References;
+  annotation (preferredView="info",Documentation(info="<html>
+
+<table border=\"0\" cellspacing=\"0\" cellpadding=\"2\">
+
+  <tr>
+    <td>[Fischer1999]</td>
+    <td>R. Fischer, <em>Elektrische Maschinen</em> (in German), 10th edition, Hanser-Verlag, 1999</td>
+  </tr>
+
```

Comment:
The 16th edition is available as open access from https://www.hanser-elibrary.com/isbn/9783446438132. Should we update and add the link?

Reply:
Only the table of contents and conclusions are free...

Reply:
Right, no open access (only checked the first chapter). Still we could add the link to the book, similarly as for @mtiller's book with costs.

Reply:
OK, I will add the actual edition of the book and the link to the publisher's web site.